From: Tollef Fog Heen <tfheen@err.no>
Date: Tue, 5 Jun 2012 20:59:36 +0200
Subject: Bring tmpfiles.d/tmp.conf in line with Debian defaults

In Ubuntu, use 30d cleanup time for /tmp, in addition to cleaning at boot.

Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/2019026
Forwarded: no, 30d cleanup is Ubuntu specific

Closes: #675422
---
 tmpfiles.d/tmp.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tmpfiles.d/tmp.conf b/tmpfiles.d/tmp.conf
index fe5225d..d029ffc 100644
--- a/tmpfiles.d/tmp.conf
+++ b/tmpfiles.d/tmp.conf
@@ -8,5 +8,5 @@
 # See tmpfiles.d(5) for details
 
 # Clear tmp directories separately, to make them easier to override
-q /tmp 1777 root root 10d
-q /var/tmp 1777 root root 30d
+D /tmp 1777 root root 30d
+#q /var/tmp 1777 root root 30d
